### DISCUSSION TITLE
The script has an unsupported MIME type ('text/plain'). in PWA

### DISCUSSION THREAD

**Discussion by ghosteye1** [quasarframework/quasar#10385] [CLI - PWA mode]
Hi guys, I added pwa support using 
$ quasar mode add pwa
$ quasar build -m pwa

After deploying to serve I'm getting 
**The script has an unsupported MIME type ('text/plain').**
error in the service-worker.js. I guess due to that reason in the App manifest I'm getting **no matching service worker detected**. Plz help me.

**Comment by ontwikkelfabriek** [comment]
@ghosteye1 

Assuming the app runs fine in dev mode, the problem is probably on your server. 
Are you using **history** or **hash** mode for your router? ( see quasar.conf.js)

**Reply by ghosteye1** [reply]
It's '**history**' mode. Yes app running fine in production server too. Only service worker not registering due to above issue

**Comment by RamonDonadeu** [comment]
Hi! Did you solve this? I've been stuck on this for a day :(
